### PR TITLE
Intro to Coding - intro slides and update to events landing page

### PR DESCRIPTION
**Pull request by kldickenson**
These are ready for Chapter Leader review.
--
Karen

### CODE REVIEW

File: events/0116_gc_intro-coding.html [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+                    <p><small>6:30pm - 8pm &nbsp; | &nbsp; 212 S. 5th Ave</small></p>
+                    <aside class="notes"></aside>
+                </div>
+                 <div class="box">
+                    <h3 class="event__title box--small green"><small><span class="blue">Intro to Ruby Workshop</span><br />4-Session Workshop</small></h3>
+                    <h3 class="event__title"><small>On Tuesdays starting 2/2 @ Nutshell</small></h3>
+                    <p><small>6:00pm - 9pm &nbsp; | &nbsp; 212 S. 5th Ave</small></p>
+                   <aside class="notes"></aside>
+                </div>
+   -->         </section>
```
Comment (jewlofthelotus) [inline events/0116_gc_intro-coding.html]: @kldickenson any particular reason you commented these events out?

File: events/0116_gc_intro-coding.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+                <h3>Sponsors!</h3>
+                <img src="images/sponsors-2015.3.png" class="img--bare box--small" />
+            </section>
+
+            <!-- Sponsor Spotlight -->
+            <section>
+                <h3>Sponsor Spotlight!</h3>
+                <img src="images/barracuda.jpg" class="img--bare box" width="400" />
+                <img src="images/tek-systems.jpg" class="img--bare box" width="400" />
+            </section>
```
Comment (jewlofthelotus) [inline events/0116_gc_intro-coding.html]: @kldickenson let's break this out into 2 slides with 1 logo each. Barracuda first, then Tek Systems

File: events/0116_gc_intro-coding.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+    <div class="reveal">
+        <!-- Any section element inside of this container is displayed as a slide -->
+        <div class="slides">
+
+            <!-- Title -->
+            <section>
+                <img src="images/gdi_logo_badge.png" class="img--bare" height="450px" />
+                <div class="box--small">
+                    <h3 class="event__title"><small><span class="green">Workshop Presented by Grand Circus</span></small><br/>Introduction to Coding</h3>
+                    <p><small>Maddy Andrade &nbsp; | &nbsp; @GrandCircusCo &nbsp; | &nbsp; @gdiannarbor &nbsp; | &nbsp; #GDIA2 &nbsp;</small></p>
```
Comment (jewlofthelotus) [inline events/0116_gc_intro-coding.html]: @kldickenson if we can make it fit, let's add a twitter handle for Maddy

**Comment by jewlofthelotus** [comment]
Looking good, @kldickenson - just left a few small items to address. Thanks!

File: events/0116_gc_intro-coding.html (hunk 1 above)
Comment (kldickenson) [inline events/0116_gc_intro-coding.html]: They didn't fit on the single page. I'll update to have a second page of GDI events.

Comment (jewlofthelotus) [inline events/0116_gc_intro-coding.html]: Ah sorry, I didn't notice how you had it set up. Each GDI event should be on its own slide



-- 

Julie Cameron

@jewlofthelotus

File: events/0116_gc_intro-coding.html (hunk 3 above)
Comment (kldickenson) [inline events/0116_gc_intro-coding.html]: It fits. Will be in next commit.

File: events/0116_gc_intro-coding.html (hunk 2 above)
Comment (kldickenson) [inline events/0116_gc_intro-coding.html]: Done and is part of next commit.

**Pull request merged**